docs/getting-started/install-elastic-endpoint.asciidoc

```diff
@@ -0,0 +1,100 @@
+
+The following message appears during installation:
+
+--
+image::images/install-endpoint/system-ext-blocked.png[]
+--
+
+. Click *Open Security Preferences*.
+. In the lower-left corner of the *Security & Privacy* pane, click the *Lock button*, then enter your credentials to authenticate.
++
+--
+image::images/install-endpoint/lock-button.png[]
++
+. Click *Allow* to allow the {elastic-endpoint} system extension to load.
+--
+image::images/install-endpoint/allow-system-ext.png[]
+
+[discrete]
+[[allow-filter-content]]
+== Approve network content filtering for {elastic-endpoint}
+
+After successfully loading the {elastic-endpoint} system extension,  an additional message appears, asking to allow {elastic-endpoint} to filter network content.
+
+--
+image::/images/install-endpoint/filter-network-content.png[
+--
+
+* Click *Allow* to enable content filtering for the {elastic-endpoint} system extension. Without this approval, {elastic-endpoint} cannot receive network events, and, therefore, cannot enable network-related features such as <<host-isolation-ov, host isolation>>.
+
+[discrete]
+[[enable-fda-endpoint]]
+== Enable Full Disk Access for {elastic-endpoint}
+
+{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {endpoint-sec} framework and to protect your network from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. To enable Full Disk Access, you must manually approve {elastic-endpoint}. For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel system extension>>.
+
+NOTE: The following instructions apply to {elastic-endpoint} only. To see Full Disk Access requirements for the Endgame sensor, refer to <will insert topic when added to TOC.>
+
+. Open the **System Preferences** application.
++
+. Select **Security and Privacy**.
++
+--
+image::images/fda/sec-privacy-pane.png[]
+--
+. On the *Security and Privacy* pane, select the **Privacy** tab.
++
+. From the left pane, select **Full Disk Access**.
++
+--
+image::images/fda/select-fda.png[Select Full Disk Access]
+--
++
+. In the lower-left corner of the pane, click the **Lock button**, then enter your credentials to authenticate.
++
+. Ensure that `ElasticEndpoint` AND `co.elastic.systemextension` are checked to properly enable Full Disk Access.
++
+NOTE: If the endpoint is running version 7.16 or earlier of {elastic-endpoint}, ensure that `elastic-endpoint` AND `co.elastic.systemextension` are checked to properly enable Full Disk Access.
++
+--
+image::images/fda/fda-7-16.png[]
+--
+
+[discrete]
+[[kernel-extension-approval]]
```

Comment:
If we are going to include this in the elastic endpoint install document, we need to be explicit in that this only applies for 7.15 and below. Since this document is for 8.0, should we just drop this section entirely? CCing @ferullo

Reply:
Removing this from the integration topic.